src/lidar/MainVis.cpp

```diff
@@ -109,8 +102,8 @@ int runRPLidar(unsigned long baudrate) {
 static void help() {
 	printf("Lidar Visualizer. \n"
 		"Usage: \n"
-		"     -l=<lidar_num>         # 1=hokyuo, 2=rplidar"
-		"     -b=<baudrate>          # set baudrate, defaults to 115200"
+		"     -l=<lidar_num>         # 1=hokyuo, 2=rplidar \n"
+		"     -b=<baudrate>          # set baudrate, defaults to 115200 \n"
```

Comment:
This help message implies that you can use the flags `-l` and `-b` with the program, but is it actually parsing these flags? The code in `main()` seems to imply that it is just getting the second and third positional arguments, which is fine, but we should probably change the help message to specify this. 
`Usage: lidar_vis <lidar_num> <baud_rate>` would probably work fine.

Also, rather than using a `lidar_num`, it might be more intuitive to use pass a `lidar_type` as a string, either `"hokuyo"` or `"rplidar"`; you can construct a string with `std::string lidar_type(argv[1])` and then just check `lidar_type == "hokuyo"`, etc.